testRenamePackage.sh

```diff
@@ -4,6 +4,6 @@ cd ..
 rm -rf android-starter-project-copy
 cp -R android-starter-project android-starter-project-copy
 cd android-starter-project-copy
-./renamePackage.sh com.demo.mobile
+python renamePackage.py com.demo.mobile
```

Comment:
Actually...would that not work for Windows?  Not sure how things work there anymore...